index.js

```diff
@@ -1117,4 +1136,9 @@ exports.main = function() {
     if(typeof simpleStorage.seenUserData === 'undefined') {
         modifyNewtab();
     }
+
+    var resetPreloadTimer = timers.setTimeout(function() {
```

Comment:
Probably should've noted here that this is the failsafe which should reset the preference after 24 hours if it isn't already reset.